tests/test_main.py

```
@@ -12,0 +1,326 @@
+
+        main_module.cleanup_old_files_main(target_date_for_cleanup, dry_run=False)
+
+        self.assertEqual(mock_delete.call_count, 3)
+        mock_delete.assert_any_call("2023-01-01_newspaper.pdf", dry_run=False)
+        mock_delete.assert_any_call("2023-01-02_thumbnail.jpg", dry_run=False)
+        mock_delete.assert_any_call("2023-01-03_newspaper.html", dry_run=False)
+    
+    @patch('main.storage.list_storage_files')
+    @patch('main.storage.get_file_url')
+    def test_get_past_papers_from_storage_logic(self, mock_get_url, mock_list_files):
+        def side_effect_config_get(key_tuple, default=None):
```

Ruff (0.11.9)

Missing return type annotation for private function `side_effect_config_get`

(ANN202)